CHANGES.rst

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+0.6.2 (unreleased)
+----------------
```

Comment:
My gut feeling says to suggest a larger version bump (say, to 0.7.0?).  I think it's mostly because of the change in dependencies (ZODB3 -> ZODB).